spec/system/custom_sidebar_sections_spec.rb

```diff
@@ -292,6 +293,7 @@
     sidebar.edit_custom_section("My section")
 
     section_modal.delete
+    expect(section_modal).to have_text("Are you sure you want to delete this section?")
```

Comment:
Should use `I18n.t` for the text comparisons please; I know everyone has a different opinion on this and this is mine :laughing:

Reply:
Those translations are on client.yml and don't know if I can easily access them from system specs

Reply:
@lis2 you can -- for future reference you just need to prefix with `js.` so e.g. `I18n.t("js.some.translation.path")`